third_party/docfx/templates/devsite/partials/uref/class.tmpl.partial

```diff
@@ -83,6 +83,105 @@
   </tbody>
 </table>
 {{/exceptions.0}}
+{{#arguments.0}}
+<div class="markdown level0 summary">

```

Comment:
Does these `markdown` classes work? The existing ones seem to _only_ have content in them, no additional HTML.

Reply:
afaict the `markdown` class does nothing. removing